src/styles/styles.css

```diff
@@ -0,0 +1,316 @@
+@charset "UTF-8";
+
+/* VARIABLES HEADER */
+
+/* VARIABLES CARD */
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` property is usually applied to all elements in the document. Consider using a universal selector (*) to apply this property to all elements. This will ensure that padding and border are included in an element's total width and height.